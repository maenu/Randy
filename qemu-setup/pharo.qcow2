version https://git-lfs.github.com/spec/v1
oid sha256:80812dbb753bada9cbadc59f07c93b155031826541f4131cdea583e2807631de
size 110821376
