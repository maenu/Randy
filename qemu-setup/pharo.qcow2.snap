version https://git-lfs.github.com/spec/v1
oid sha256:37888604c0a51f14eeb1d93d0f48b64626d5df24c4acf9d0d39c1642cb6caf49
size 180027392
